### PR TITLE
project: Release v0.1.0-alpha.11

### PR DESCRIPTION
**Pull request by sreis**
I've already built and pushed the docker image. After merging this we need to create and push the tag.

### CODE REVIEW

File: RELEASE-NOTES.adoc [hunk 1]
```diff
@@ -19,6 +19,16 @@ endif::[]
 == Table of Contents
 toc::[]
 
+== v0.1.0-alpha.11
+
+=== Bug fixes
+
+* Fix a bug which caused `dklb` to create an invalid DC/OS secret (https://github.com/mesosphere/dklb/pull/43[#43]).
```
Comment (jimmidyson) [inline RELEASE-NOTES.adoc]:
```suggestion
* Fix DC/OS secret names when Kubernetes is deployed in a DC/OS folder (https://github.com/mesosphere/dklb/pull/43[#43]).
```

File: RELEASE-NOTES.adoc [hunk 2]
```diff
@@ -19,6 +19,16 @@ endif::[]
...
+
+=== Documentation
+
+* Add command to cleanup Kubernetes secret in TLS for Ingress example https://github.com/mesosphere/dklb/pull/43[#43]).
```
Comment (jimmidyson) [inline RELEASE-NOTES.adoc]:
```suggestion
* Add command to cleanup Kubernetes secret in TLS for Ingress example (https://github.com/mesosphere/dklb/pull/43[#43]).
```

Comment (sreis) [reply RELEASE-NOTES.adoc]: Done.

File: RELEASE-NOTES.adoc (hunk 1 above)
Comment (sreis) [reply RELEASE-NOTES.adoc]: Done.

**Pull request merged**